docs/en/query_language/dicts/external_dicts_dict_hierarchical.md

```diff
@@ -0,0 +1,35 @@
+# Hierarchical Dictionaries
+
+ClickHouse supports hierarchical dictionaries with a [numeric key](external_dicts_dict_structure.md#ext_dict-numeric-key).
+
+Look at the following hierarchical structure:
+
+```text
+0 (Common ancestor)
+│
+├── 1 (Russia)
+│   │
+│   └── 2 (Moscow) 
+│       │
+│       └── 3 (Center)  
+│
+└── 4 (Great Britain)
+    │
+    └── 5 (London)
+```
+
+This hierarchy can be expressed as the following dictionary table.
+
+Key | Ancestor key | Name
```

Comment:
I believe it is easier for everyone to use parent-child terminology.

Reply:
Ok.